packages/dashboard-v2/gatsby-ssr.js

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import "@fontsource/sora/300.css"; // light
+import "@fontsource/sora/400.css"; // normal
+import "@fontsource/sora/500.css"; // medium
+import "@fontsource/sora/600.css"; // semibold
+import "@fontsource/source-sans-pro/400.css"; // normal
+import "@fontsource/source-sans-pro/600.css"; // semibold
+import "./src/styles/global.css";
+
+export function wrapPageElement({ element, props }) {
+  const Layout = element.type.Layout ?? React.Fragment;
+  return <Layout {...props}>{element}</Layout>;
+}
```

Comment:
seems you've added this file again in this pr - what's up with that, is the build failing without this file?

Reply:
Yeah, I mentioned it in the PR description:
> * Adds back gatsby-ssr.js file -- it's needed for `gatsby build` followed by `gatsby serve` flow (seems like SSR is enabled with `gatsby serve`).

Reply:
I just locally removed this file and ran a build and it passed. Can you double check please?